script.js

```diff
@@ -33,6 +35,11 @@ const cartoSheetSyncTable =
 const cartoCountiesURI = createCountiesCartoURI();
 const cartoStatesURI = createStatesCartoURI();
 
+// colorScale comes from this ColorBrewer url:
+// https://colorbrewer2.org/#type=sequential&scheme=YlGn&n=7
+const fillColorScale = ["","#d9f0a3","#78c679","#238443"];
```

Comment:
@jimboweb we should probably have a fallback color in case a geography is missing a score in the "range" field / property. In #54 @river-honer is using `#808080` for the fill in such scenarios. This would help researchers and others identify where a geography is missing a score so it can be fixed. Otherwise not having a fill and stroke (because an unmatched array value will return `undefined` will result in the geography not rendering on the map at all, which would not be good!